src/container.ts

```diff
@@ -72,19 +72,32 @@ container.register({
   samlIssuer: awilix.asValue(SAML_ISSUER)
 });
 
+// Redirection urls
+container.register({
+  clientErrorRedirectionUrl: awilix.asValue(
+    process.env.CLIENT_ERROR_REDIRECTION_URL || "/error.html"
+  ),
+  clientLoginRedirectionUrl: awilix.asValue(
+    process.env.CLIENT_REDIRECTION_URL || "/login"
+  ),
+  clientProfileRedirectionUrl: awilix.asValue(
+    process.env.CLIENT_REDIRECTION_URL || "/profile.html?token={token}"
```

Comment:
also, to avoid referencing the `{token}` string in other places (i.e. `AuthenticationController#acs`) you could isolate the responsibility of interpolating this string here by changing the type of `clientProfileRedirectionUrl` from a `string` to a `string => string`